data/man/en.rst

```diff
@@ -221,7 +221,7 @@ To explicitly specify the formats in the above example, append filenames with
 appropriate **-f** options::
 
     gramps -i file1.ged -f gedcom -i file2.tgz -f gramps-pkg \
-    -i ~/db3.gramps -f gramps-xml -i file4.wft -f wft -a check
+    -i ~/db3.gramps -f gramps -i file4.wft -f wft -a check
```

Comment:
As an argument, changing to `-f gramps` is correct. All the other instances of `gramps-xml` on this page refer to the file format which is ["Gramps XML" format](https://www.gramps-project.org/wiki/index.php/Gramps_XML) so I believe it is OK to leave them as-is. However I would defer to someone who has some historical knowledge of the change. @Nick-Hall or @emyoulation or...?